mysql/datadog_checks/mysql/data/conf.yaml.example

```diff
@@ -21,230 +29,280 @@ instances:
+        ## MySQL integration tile in the Datadog Web UI for further instructions.
+        #
+        # extra_performance_metrics: true
+
+    ## @param tags - list of strings - optional
+    ## A list of tags to attach to every metric and service check emitted by this instance.
+    ##
+    ## Learn more about tagging at https://docs.datadoghq.com/tagging
+    #
+    # tags:
+    #   - <KEY_1>:<VALUE_1>
+    #   - <KEY_2>:<VALUE_2>
+
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Overrides any `service` defined in the `init_config` section.
+    #
+    # service: <SERVICE>
+
+    ## @param min_collection_interval - number - optional - default: 15
+    ## This changes the collection interval of the check. For more information, see:
+    ## https://docs.datadoghq.com/developers/write_agent_check/#collection-interval
+    #
+    # min_collection_interval: 15
+
+    ## @param empty_default_hostname - boolean - optional - default: false
+    ## This forces the check to send metrics with no hostname.
+    ##
+    ## This is useful for cluster-level checks.
+    #
+    # empty_default_hostname: false
+
+## Log Section
 ##
-## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
-## port / path / channel_path - mandatory - Set port if type is tcp or udp. Set path if type is file. Set channel_path if type is windows_event
-## service - mandatory - Name of the service that generated the log
-## source  - mandatory - Attribute that defines which Integration sent the logs
-## tags: - optional - Add tags to the collected logs
+## type - required - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel_path - required - Set port if type is tcp or udp.
+##                                         Set path if type is file.
+##                                         Set channel_path if type is windows_event.
+## source  - required - Attribute that defines which Integration sent the logs
+## service - required - The name of the service that generates the log.
+##                      Overrides any `service` defined in the `init_config` section.
+## tags - optional - Add tags to the collected logs
 ##
 ## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
 #
 # logs:
 #   - type: file
-#     path: "<ERROR_LOG_FILE_PATH>"
+#     path: <GENERAL_LOG_FILE_PATH>
 #     source: mysql
-#     service: "<SERVICE_NAME>"
-#
+#     service: <SERVICE>
 #   - type: file
-#     path: "<SLOW_QUERY_LOG_FILE_PATH>"
+#     path: <ERROR_LOG_FILE_PATH>
 #     source: mysql
-#     service: "<SERVICE_NAME>"
-#     log_processing_rules:
-#       - type: multi_line
-#         name: new_slow_query_log_entry
-#         pattern: "# Time:"
-#         # If mysqld was started with `--log-short-format`, use:
-#         # pattern: "# Query_time:"
```

Comment:
Can we keep those comments, seems useful ?
```
#         # If mysqld was started with `--log-short-format`, use:
#         # pattern: "# Query_time:"
```

Reply:
Yes good catch, I do want to keep it but `yaml.safe_load` strips comments so they're not conserved when ddev calls `yaml.dumps()` on the dict 😕 

I could do `\# comment` but that's output as-is in the config, so not sure how to figure this out…

```yaml
#       \# If mysqld was started with `--log-short-format`, use: null
#       \# pattern: '# Query_time:'

```

Reply:
I think the usual workaround is the add the comment in the description of the `logs:` section.

Reply:
How would you go about that? The `description` is provided by the `logs.yaml` template, I don't think I can easily append lines to it. An alternative is to copy-paste the existing description, but it's not great…

Reply:
Yes, I was thinking about overriding, but as you mentioned this will can lead to duplication.

A possible solution is to provide some variable that reference parent content (a new feature for config spec).

```
      overrides:
        host.description: |
          {{parent_content}}
          My new content
```

Reply:
That said, if I have to choose between duplication or keeping a useful comment, I would prefer the later.

Reply:
OK, so for now I added an extra entry to `log_processing_rules` for the `# Query_time:` pattern. Hopefully the `name` makes it clear enough that it's meant for `mysqld --short-log-format`. Because even if not, I don't think it's a problem if users leave it there (the rule won't match if mysqld isn't using that option).

Reply:
awesome 👍